scout/commands/load/panel.py

```diff
@@ -101,53 +105,3 @@ def panel(
     except Exception as err:
         LOG.warning(err)
         raise click.Abort()
-
-    return
-
-
-def _panel_omim(adapter, genemap2, mim2genes, api_key, institute):
```

Comment:
I've just moved this to a better place, together with similar panel loading helpers